imagersite/imager_images/templates/album.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+{% load thumbnail %}
+{% block content %}
+
+<div class='imager-album'>
+
+<h2>{{ user.get_full_name }}</h2>
+
+<h2>{{ album.title }}
+
+
+<h3>All Photos</h3>
+<div class='all_photos'>
+{% for photo in album.photos.all %}
+
+    <h3 class='photo_title'>{{ photo.title }}</h3>
+    {% thumbnail photo.file "180x180" as im %}
+    <a href="{% url 'photo_detail' photo.id %}"><img src='{{ MEDIA_ROOT }}{{ im.url }}'/></a>
+    {% endthumbnail %}
+
+{% endfor %}
+{% cycle "" "" "<br/>"%}
+</div>
+
+<h3>Description</h3>
+<p>{{ album.description }}</p>
+<p><strong>Uploaded:</strong> {{ album.date_uploaded|date:"D d M Y" }}</p>
+<p><strong>Modified:</strong> {{ album.date_modified|date:"D d M Y" }}</p>
+<p><strong>Published:</strong> {{ album.date_published|date:"D d M Y" }}</p>
```

Comment:
This would be a good place for something like this:

```html
{% with "D d M Y" as date_format %}
<p><strong>Uploaded:</strong> {{ album.date_uploaded|date:date_format }}</p>
<p><strong>Modified:</strong> {{ album.date_modified|date:date_format }}</p>
<p><strong>Published:</strong> {{ album.date_published|date:date_format }}</p>
{% endwith %}
```
Then if you decide down the road to change the format, you only need to do so in one place.